### ISSUE TITLE
things on summary table should not be links if that same thing is already a selection criteria

### ISSUE THREAD

**Issue by phirstgemini** [GeminiDRSoftware/FitsStorage#69]
need to avoid encouraging the blah/blah/blah/blah thing.

**Comment by phirstgemini** [comment]
Initial analysis suggests this could be done at least to 90% with a macro in summary_entries.macro.
Would be nice to refactor / clean up that whole area, but not for 3.3

**Comment by phirstgemini** [comment]
```
2024-10-05 08:06:26,998 1635660:queue:96 DEBUG: Popped id 102392 - N20241005S0292.fits from exportqueue
2024-10-05 08:06:27,003 1635660:service_export_queue:119 INFO: Exporting N20241005S0292.fits - 102392 (694 on queue)
2024-10-05 08:06:27,003 1635660:exporter:106 DEBUG: Export N20241005S0292.fits to https://archive.gemini.edu
2024-10-05 08:06:27,097 1635660:connectionpool:546 DEBUG: https://archive.gemini.edu:443 "GET /jsonfilelist/present/filename=N20241005S0292.fits HTTP
/1.1" 200 None
2024-10-05 08:06:27,097 1635660:exporter:420 DEBUG: Got destination file info for filename N20241005S0292.fits: md5 93ae0e07367ccf3b64c321fe077367b6,
 pending_ingest: True
2024-10-05 08:06:27,097 1635660:exporter:118 DEBUG: Sucessfully got destination file info for N20241005S0292.fits
2024-10-05 08:06:27,208 1635660:queue:96 DEBUG: Popped id 102394 - N20241005S0291.fits from exportqueue
```

**Comment by phirstgemini** [comment]
Hopefully fixed in 46e5ba76
When file was pending ingest on destination, we were setting eqe.after to delay the export but not setting eqe.inprogress to false before returning.
Complicated by logger apparently silently failing due to being passed a string argument to a field erroneously flagged as %d.

**Comment by phirstgemini** [comment]
Fixed.. Well, they're still links, but they redirect back to a normalized URL, so we don't get the link chaining effect.

**Comment by phirstgemini** [comment]
Included in 3.3
